.github/wildfly-bot.yml

```diff
@@ -280,6 +280,6 @@ wildfly:
   format:
     description:
       regexes:
-        - pattern: "JIRA:\\s+https://issues.redhat.com/browse/WFLY-\\d+|https://issues.redhat.com/browse/WFLY-\\d+"
-          message: "The description must contain a link to the JIRA issue \n(JIRA: https://issues.redhat.com/browse/WFLY-XYZ or https://issues.redhat.com/browse/WFLY-XYZ)"
+        - pattern: "https://issues.redhat.com/browse/WFLY-\\d+|https://issues.jboss.org/browse/WFLY-\\d+"
+          message: "The description must contain a link to the JIRA issue \n(https://issues.redhat.com/browse/WFLY-XYZ or https://issues.jboss.org/browse/WFLY-XYZ)"
```

Comment:
We should leave off the "org https://issues.jboss.org/browse/WFLY-XYZ)" IMO. We should be using issues.redhat.com as that is what issues.jboss.org redirects to. I think allowing it in the pattern is fine though.

Reply:
well, ok, but still it seems weird that it is still a valid link and we fail the format check

Reply:
Sorry, I realize it's merged now. What I meant was IMO it would be okay keep it from failing with the pattern, but we shouldn't encourage the usage in the error message.

That said, the only way I could see the jboss.org URL being used is if it's manually typed.